TrueCraft.Launcher/Views/MultiplayerView.cs

```diff
@@ -131,7 +143,10 @@ public MultiplayerView(LauncherWindow window)
             foreach (var server in UserSettings.Local.FavoriteServers)
             {
                 var row = ServerListStore.AddRow();
-                using (var stream = Assembly.GetExecutingAssembly().GetManifestResourceStream("TrueCraft.Launcher.Content.default-server-icon.png"))
+                using (
+                    var stream =
+                        Assembly.GetExecutingAssembly()
+                            .GetManifestResourceStream("TrueCraft.Launcher.Content.default-server-icon.png"))
```

Comment:
Look dude, coding style will be enforced on this project. You do not get to change that. Even so, I can't reject or accept only parts of the pull request. I would have left it open and given you a chance to undo the coding style changes, but I rejected it because the way you went about implementing this feature is wrong. Take it easy, dude.